searcher/src/main.rs

```diff
@@ -157,6 +157,8 @@ fn search<P: AsRef<Path>>(
         error!("Unable to join internal thread: {:?}", e);
     }
 
+    let i: usize = processed_sigs.fetch_max(0, Ordering::SeqCst);
```

Comment:
```suggestion
    let i: usize = processed_sigs.into_inner()
```
would have worked too (it is initialized with `0`, and it is not used after this point)
https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.into_inner